apps/addons/templates/addons/impala/details.html

```diff
@@ -199,7 +199,7 @@ <h4 class="author">{{ _('by') }} {{ users_list(addon.listed_authors) }}</h4>
   {% if version %}
     <ul>
       <li>{{ _('Version {0}')|f(version.version) }} <a class="scrollto" href="#detail-relnotes">{{ _('Info') }}</a></li>
-      <li>{{ version.created|datetime }}</li>
+      <li>{% trans %}Last Updated{% endtrans %}: {{ version.created|datetime }}</li>
```

Comment:
The `:` needs to be inside the `{% trans %}` tag, as the punctuation and whitespace rules differ between languages.

For example, there's always a non-breaking space just before a `:` in french.